### PR TITLE
s/PA/PDA/g

### PR DESCRIPTION
**Pull request by cjpatton**

### CODE REVIEW

File: draft-pda-protocol.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ The protocol is executed by a large set of clients and a small set of servers.
 We call the servers the *aggregators*. Each client's input to the protocol is a
 set of measurements (e.g., counts of some user behavior). Given the input set
 of measurements `x_1, ..., x_n` held by `n` users, the goal of a
-*private aggregation (PA) protocol* is to compute `y = F(x_1, ..., x_n)` for
+*private aggregation (PDA) protocol* is to compute `y = F(x_1, ..., x_n)` for
```
Comment (tgeoghegan) [inline draft-pda-protocol.md:129]:
```suggestion
*private data aggregation (PDA) protocol* is to compute `y = F(x_1, ..., x_n)` for
```

**Review by tgeoghegan — APPROVED** [review]
LGTM modulo one nit

File: draft-pda-protocol.md (hunk 1 above)
Comment (cjpatton) [reply draft-pda-protocol.md:129]: Good catch

**Comment by cjpatton** [comment]
Squashed.

**Pull request merged**